Core/Src/mainTask.c

```diff
@@ -107,7 +104,7 @@ void runMain()
 			{
 				printf("Balancing Enabled: FALSE\n");
 			}
-			balancePack(numBmbs, balancingEnabled);
+			balancePack(balancingEnabled);
```

Comment:
`balancingEnabled` argument shouldn't be used. I'm confused why it doesn't throw a compile time warning...